static/src/javascripts/bootstraps/commercial.js

```diff
@@ -28,24 +28,24 @@ import { initCheckDispatcher } from 'commercial/modules/check-dispatcher';
 import commentAdverts from 'commercial/modules/comment-adverts';
 
 const commercialModules: Array<Array<any>> = [
```

Comment:
This changes the ordering important here a bit. Does anyone know if that's a problem?